### ISSUE TITLE
[ENTRY] Quantum CV RNN

### ISSUE THREAD

**Issue by AndreyYashkin** [XanaduAI/QHack2021#49]
### Team Name: 

QuantumMadness

### Project Description: 

This project is an attempt to build a quantum recurrent neural network using the continuous variable quantum model.  As sources of inspiration a [quantum qubit based RNN architecture](https://arxiv.org/pdf/2006.14619.pdf) which allows to reuse input wire, the quantum neural network [tutorial from strawberryfields.ai](https://strawberryfields.ai/photonics/demos/run_quantum_neural_network.html)  and the Elman RNN architecture were taken.

Due to the hight computational cost for the simulation of Fock space only primitive tests can be accomplished. RNN was tested to be suitable for learning XOR function from sequences of zeros and ones. From an investigation done so far it seems to be more sensitive to learning rate than its classic analog. Also the RNN circuit is quite long what may lead to a situation when a state created by the circuit is physically incorrect due to computational mistakes led by limited Fock dimensional size.

### Presentation: 

Check README.md

### Source code: 

[repo](https://github.com/nsu-ai/quantum-CV-RNN)

**Comment by AndreyYashkin** [comment]
Sorry for a late final submission. We got confused with time zones:)

**Comment by co9olguy** [comment]
Thanks for the submission! We hope you have enjoyed participating in QHack :smiley: 

We will be assessing the entries and contacting the winners separately. Winners will be publicly announced sometime in the next month.

We will also be freezing the GitHub repo as we sort through the submitted projects, so you will not be able to update this submission.
